Formula/fortio.rb

```diff
@@ -0,0 +1,36 @@
+class Fortio < Formula
+  desc "HTTP and gRPC load testing and visualization tool and server"
+  homepage "https://github.com/istio/fortio"
+  url "https://github.com/istio/fortio.git",
+      :using => :git,
+      :tag => "v0.9.0",
+      :revision => "e50943e8e525197f36f9b4f81464615c063a0a65"
```

Comment:
anyway to get the latest release/tag at the time of build? or do we need to update this for every release ? `latest_release` in git points to the latest good tag

Reply:
We can but then you:
* Experience a crash in brew build which seems like version in the tag
* Won't pass `brew audit --strict` as it expects `revision` too.
* Won't have the ability to install an earlier version and